syntax = "proto3";

message Product {
    int32 id = 1;
    string name = 2;
    string description = 3;
    float price = 4;
    int32 stock = 5;
}

message InventoryUpdate {
    int32 product_id = 1;
    int32 new_stock = 2;
}